dataplane/src/args.rs

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
+
+pub(crate) use clap::Parser;
+#[derive(Parser)]
+#[command(name = "Hedgehog Fabric Gateway dataplane")]
+#[command(version = "1.0")] // FIXME
+#[command(about = "A next-gen dataplane for next-gen fabric gateway", long_about = None)]
+pub(crate) struct CmdArgs {
+    #[arg(long, value_name = "core-id used as main", default_value_t = 2)]
+    main_lcore: u8,
+    #[arg(long, value_name = "map lcore set to cpu set")]
+    lcores: Option<String>,
+    #[arg(long, value_name = "in-memory flag", default_value_t = true)]
```

Comment:
I suspect that we should "hard code" the `--in-memory` option.

Otherwise we risk leaking truly massive amounts of hugepage memory if DPDK gets a `kill -9` or some other event which prevents it from cleaning up.

Reply:
I'm fine with that. Mind it defaults to true. But if you think that it should always be there, we can remove the option.